packages/directlinespeech/src/createAdapters.js

```diff
@@ -104,11 +109,17 @@ export default async function create({
     config = BotFrameworkConfig.fromSubscription(subscriptionKey, region);
   }
 
+  // switch to direct line endpoint on DLS service.
+  if (enableInternalHttpSupport) {
+    const endpoint = "wss://" + region + ".convai.speech.microsoft.com/directline/api/v1";
```

Comment:
```suggestion
    const endpoint = `wss://${region}.convai.speech.microsoft.com/directline/api/v1`;
```

Also, we don't use double-quote for string. Can you run Prettier on the file?